PWGJE/DataModel/JetTagging.h

```diff
@@ -120,21 +122,28 @@ JETSV_TABLES_DEF(Charged, SecondaryVertex3Prong, "3PRONG");
 JETSV_TABLES_DEF(Charged, SecondaryVertex2Prong, "2PRONG");
 
 // Defines the jet substrcuture table definition
-#define JETTAGGING_TABLE_DEF(_jet_type_, _name_, _description_)       \
-  namespace _name_##tagging                                           \
-  {                                                                   \
-    DECLARE_SOA_COLUMN(Origin, origin, int);                          \
-    DECLARE_SOA_COLUMN(JetProb, jetProb, std::vector<float>);         \
-    DECLARE_SOA_COLUMN(FlagtaggedjetIP, flagtaggedjetIP, bool);       \
-    DECLARE_SOA_COLUMN(FlagtaggedjetIPxyz, flagtaggedjetIPxyz, bool); \
-    DECLARE_SOA_COLUMN(FlagtaggedjetSV, flagtaggedjetSV, bool);       \
-    DECLARE_SOA_COLUMN(FlagtaggedjetSVxyz, flagtaggedjetSVxyz, bool); \
-  }                                                                   \
-  DECLARE_SOA_TABLE(_jet_type_##Tags, "AOD", _description_ "Tags", _name_##tagging::Origin, _name_##tagging::JetProb, _name_##tagging::FlagtaggedjetIP, _name_##tagging::FlagtaggedjetIPxyz, _name_##tagging::FlagtaggedjetSV, _name_##tagging::FlagtaggedjetSVxyz);
-
-#define JETTAGGING_TABLES_DEF(_jet_type_, _description_)                                                    \
-  JETTAGGING_TABLE_DEF(_jet_type_##Jet, _jet_type_##jet, _description_)                                     \
-  JETTAGGING_TABLE_DEF(_jet_type_##MCDetectorLevelJet, _jet_type_##mcdetectorleveljet, _description_ "MCD") \
+#define JETFLAVOURDEF_TABLE_DEF(_jet_type_, _name_, _description_) \
+  namespace _name_##flavourdef                                     \
+  {                                                                \
+    DECLARE_SOA_COLUMN(Origin, origin, int8_t);                    \
+  }                                                                \
+  DECLARE_SOA_TABLE(_jet_type_##FlavourDef, "AOD", _description_ "FlavourDef", _name_##flavourdef::Origin);
+
+#define JETTAGGING_TABLE_DEF(_jet_type_, _name_, _description_)                                                                         \
+  namespace _name_##tagging                                                                                                             \
+  {                                                                                                                                     \
+    DECLARE_SOA_COLUMN(BitTaggedjet, bitTaggedjet, uint8_t);                                                                            \
```

Comment:
this and the two below can be in a different namespace that is common. Only the dynamic coloumn needs to be in the templated namespace

Reply:
alos change to unit16_t

Reply:
or move all 4 coloumns out and put a templated dummy dynamic coloumn into the table

Reply:
I've done but I would like to check they are good ways